doc/source/whatsnew/v0.15.0.rst

```diff
@@ -754,32 +756,32 @@ Other notable API changes:
 
   .. ipython:: python
 
-     s = Series(["a", "b", "c"])
-     s.loc[0] = None
-     s
+      s = pd.Series(["a", "b", "c"])
+      s.loc[0] = None
+      s
 
   To insert a ``NaN``, you must explicitly use ``np.nan``. See the :ref:`docs <missing.inserting>`.
 
 - In prior versions, updating a pandas object inplace would not reflect in other python references to this object. (:issue:`8511`, :issue:`5104`)
 
   .. ipython:: python
 
-     s = Series([1, 2, 3])
-     s2 = s
-     s += 1.5
+      s = pd.Series([1, 2, 3])
+      s2 = s
+      s += 1.5
 
   Behavior prior to v0.15.0
 
-  .. code-block:: ipython
+  ..  code-block:: ipython
```

Comment:
I think I already said it in the previous review. All those spaces added here and next are wrong, restore the previous code (everything until the end of the file, excepting when you add `pd.`).